documentation/IDTA-01003-a/modules/ROOT/pages/IDTA-01003-a_ChangeLog.adoc

```diff
@@ -40,71 +55,130 @@ Minor changes:
+| | AASc-010 | Renamed a| renamed to AASc-3a-010
+
+
+| | AASc-3a-050 | Update a| Now `\https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIec61360/3`
+
+AASc-3a-050: If the _DataSpecificationContent_ _DataSpecificationIec61360_ is used for an element, the value of _HasDataSpecification/dataSpecification_ shall contain the external reference to the IRI of the corresponding data specification template `\https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIec61360/3`.
 |===
 
 
+
+
 == Changes V3.0.1 vs. V3.0
 
 Bugfixes:
 
-* also support deprecated template ID `\http://admin-shell.io/DataSpecificationTemplates/DataSpecificationIec61360/3/0` (https://github.com/admin-shell-io/aas-specs-iec61360/issues/4[#4])
-* removed sentence stating that the template ID is conformant to the rules for semantic IDs for data specifications as defined in Part 1 of the document series [AAS-Part1]: this is not the case but the ID will not be changed 
+* also support deprecated data specification template IDs  (https://github.com/admin-shell-io/aas-specs-iec61360/issues/4[#4], https://github.com/admin-shell-io/aas-specs-iec61360/issues/2[#2])
+
+** `\http://admin-shell.io/DataSpecificationTemplates/DataSpecificationIec61360/3/0` 
+** `\http://admin-shell.io/DataSpecificationTemplates/DataSpecificationIEC61360/3/0` 
+** `\https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIEC61360/3/0` 
+
+* For backward compatibility of future versions of this specification the ID of data specification template and value of attribute “id” of DataSpecification are now distinguished: Therefore  `\https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIec61360/3/0`  is also deprecated and `\https://admin-shell.io/DataSpecificationTemplates/DataSpecificationIec61360/3`  shall be used instead
+
+
+* corrected examples for qualifier of namespace IEC (min instead of Min for enumeration LevelType)
+
+* removed sentence stating that the template ID is conformant to the rules for semantic IDs for data specifications as defined in 
+Part 1 (IDTA-01001-3-0) of the document series: this is not the case but the ID will not be changed 
+
 * (Editorial) Constraint AASc-3a-050: external reference instead of globale reference 
+
 * (Editorial) Notes "Note: it is recommended to use a global
 reference." were updated to "Note: it is recommended to use an external
 reference." (https://github.com/admin-shell-io/aas-specs-iec61360/issues/5[#5])
-* (Editorial) Examples corrected for `IEC:DataSpecificationIec61360/levelType/Min` and `IEC:LevelType/Min` (min with small letter)
+
+
 
 .New, Changed or Removed Constraints in V3.0.1
 [width="100%",cols="7%,12%,15%,66%",options="header",]
 |===
-h|Nc h|V3.0.1 vs. V3.0 h|New, Update, Removed, Reformulated h|Comment
-| | AASc-3a-050 | Update a| External instead of global reference
+h|nc h|V3.0 vs. V3.0.1 h|New, Update, Removed, Reformulated h|Comment
```

Comment:
Why change order of versions? Shouldn't it be "V3.0.1 vs. V3.0" ?

Reply:
yes, it should

Reply:
fixed